internal/app/machined/pkg/system/services/system-containerd.go

```diff
@@ -59,8 +59,6 @@ func (c *SystemContainerd) Runner(config runtime.Configurator) (runner.Runner, e
 			"--address", constants.SystemContainerdAddress,
 			"--state", "/run/system/containerd",
 			"--root", "/run/system/lib/containerd",
-			"--config",
-			"/etc/containerd-system.toml",
```

Comment:
We have unknowingly been using the defaults, so no need for the config anymore. If we ever find a need for it we can always bring it back in pretty easily.